R/PDBHGNC.R

```diff
@@ -0,0 +1,21 @@
+#' PDBHGNC.R
```

Comment:
Nice - a data object. Now: we'll actually prefer to ;load data via fetchData(), but it's nice to see this mechanism being used correctly.